joomla/joomla-cms
# [4.0] Add extension name/id to uninstall message if extension is locked, redo of PR 28916

Pull Request for Issue # .

### Summary of Changes

This is a redo of PR #28916 . Thanks @ReLater for the implementation.

- Change lang string COM_INSTALLER_UNINSTALL_ERROR_LOCKED_EXTENSION.
- Use sprintf to display extension name and id.

### Testing Instructions
- 4.0 nightly build of today (Wednesday, 06 May 2020) or later.
- Go to extensions manager (`administrator/index.php?option=com_installer&view=manage`).
- Select many locked extensions.
- Click `Uninstall` and check the messages at the top.
- CLI: Use `php cli/joomla.php extension:remove xxx`, with `xxx`being an extension ID for a locked core extension, e.g. `Action Log - Joomla`.

### Actual result

In backend in extensions manage:
![03-05-_2020_14-23-04](https://user-images.githubusercontent.com/20780646/80914422-aa73c980-8d4b-11ea-9026-153f45a26971.jpg)

CLI:
![pr-28960_before_cli](https://user-images.githubusercontent.com/7413183/81155703-856a9b00-8f85-11ea-8f57-34b4cb8f2913.png)

### Expected result after applying PR

In backend in extensions manage:
![pr-28960_after_backend](https://user-images.githubusercontent.com/7413183/81154263-07f25b00-8f84-11ea-8927-5a947c6c0759.png)

CLI:
![pr-28960_after_cli](https://user-images.githubusercontent.com/7413183/81154626-60c1f380-8f84-11ea-84ff-a8910a2a4f69.png)
